documentation/src/main/wiki/ActionKnot.md

```diff
@@ -137,3 +137,24 @@ Adapter metadata options available. Take into consideration that Adapters are us
 | `params`                    | `JSON object`                       | &#10004;       | Default params which are sent to Adapter. |
 | `allowedRequestHeaders`     | `String`                            | &#10004;       | Array of HTTP client request headers that are allowed to be passed to Adapter. **No** request headers are allowed if not set. |
 | `allowedResponseHeaders`    | `String`                            | &#10004;       | Array of HTTP response headers that are allowed to be sent in a client response. **No** response headers are allowed if not set. |
+
+### Vert.x Event Bus delivery options
+
+While HTTP request processing, Action Knot calls Adapter using 
```

Comment:
`During HTTP request processing, Knot communicates with Adapter over the Vert.x event bus. Low level aspects of that communication can be configured using Vert.x Delivery Options object. The caller of the Adapter, such as Action or Service Knot, can customise default settings by providing a deliveryOptions field in the config section of its configuration.

Comment: Can we move a description of setting up delivery option to one place ? e.g. on Knot documentation page ? it would be hard to manage doc if it's in multiple places

Reply:
Descriptions are similar, but not the same. I thought about that but we lose the information where (between what modules) the communication occurs.

Reply:
Done.